tests/src/OneLogin/Saml2/UtilsTest.php

```diff
@@ -840,7 +843,7 @@ public function testGenerateNameIdWithSPNameQualifier()
 
         $expectedNameId = '<saml:NameID SPNameQualifier="http://stuff.com/endpoints/metadata.php" Format="urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified">ONELOGIN_ce998811003f4e60f8b07a311dc641621379cfde</saml:NameID>';
 
-        $this->assertEquals($expectedNameId, $nameId);
+        $this->assertEquals($nameId, $expectedNameId);
```

Comment:
if I remember correctly expected should be in the 1st place
````
$this->assertEquals($expectedNameId, $nameId)
````

Reply:
yes